server/storage/interface.go

```diff
@@ -33,17 +33,17 @@ type MetaStore interface {
 	// GetCurrent returns the modification date and data part of the metadata for
 	// the latest version of the given GUN and role.  If there is no data for
 	// the given GUN and role, an error is returned.
-	GetCurrent(gun data.GUN, tufRole data.RoleName) (created *time.Time, data []byte, err error)
+	GetCurrent(gun data.GUN, tufRole data.RoleName, channels ...*Channel) (created *time.Time, data []byte, err error)
```

Comment:
It seems like in all the implementations, if multiple channels are passed, only the data in the first channel the data is found in is passed (and the channels are checked in order of preference).  Should this behavior be documented as a requirement in the interface, and a general test be added (maybe in `server/storage/storage_test.go`) to make sure that even if there is a TUF file with the same GUN and role is in multiple channels, this function will only return the data from the first?

Similarly for `GetVersion`.  I guess for `GetChecksum` it doesn't really matter, since if it's the same checksum, it doesn't matter which channel it's in because all the data will be the same, although the creation time may be different.  If the creation time matters, then perhaps a similar test can be added for `GetChecksum`?